tests/clientEncryption/clientEncryption-createDataKey-001.phpt

```diff
@@ -0,0 +1,28 @@
+--TEST--
+MongoDB\Driver\clientEncryption::createDataKey()
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php skip_if_not_libmongocrypt(); ?>
+<?php skip_if_not_live(); ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+$key = base64_decode('Mng0NCt4ZHVUYUJCa1kxNkVyNUR1QURhZ2h2UzR2d2RrZzh0cFBwM3R6NmdWMDFBMUN3YkQ5aXRRMkhGRGdQV09wOGVNYUMxT2k3NjZKelhaQmRCZGJkTXVyZG9uSjFk');
+
+$manager = new MongoDB\Driver\Manager(URI);
+$clientEncryption = $manager->clientEncryption(['keyVaultNamespace' => 'default.keys', 'kmsProviders' => ['local' => ['key' => new MongoDB\BSON\Binary($key, 0)]]]);
+
+var_dump($clientEncryption->createDataKey('local'));
+
+?>
+===DONE===
+<?php exit(0); ?>
+--EXPECTF--
+object(MongoDB\BSON\Binary)#%d (%d) {
+  ["data"]=>
+  string(16) "%a"
```

Comment:
Since the spec explicitly calls for a binary with subtype 4 (UUID) to be returned, I decided to add that to the test. I can also just assert against the return value being a binary or even anything if you prefer

Reply:
Oh, I'm fine with the expected output and asserting the UUID subtype on the line below. I was mainly inquiring about why we use `%a` here. Reading this again, I think that question has been answered so I'll resolve.